rotkehlchen/assets/utils.py

```diff
@@ -30,6 +31,17 @@
 logger = logging.getLogger(__name__)
 log = RotkehlchenLogsAdapter(logger)
 
+SPAM_ASSET_REGEX = r"""
+    https:// |                          # Matches 'https://'
```

Comment:
This would match https only in the beginning right? Or not?  Same with Claim or visit?

Need more tests as I wrote in discord